content/01.abstract.md

```diff
@@ -1,11 +1,10 @@
 ## Abstract {.page_break_before}
 
-Deep learning, which describes a class of machine learning algorithms, has recently showed impressive results across a variety of domains.
-Biology and medicine are data rich, but the data are complex and often ill-understood.
-Problems of this nature may be particularly well-suited to deep learning techniques.
-We examine applications of deep learning to a variety of biomedical problems---patient classification, fundamental biological processes, and treatment of patients---and discuss whether deep learning will transform these tasks or if the biomedical sphere poses unique challenges.
-We find that deep learning has yet to revolutionize or definitively resolve any of these problems, but promising advances have been made on the prior state of the art.
-Even when improvement over a previous baseline has been modest, we have seen signs that deep learning methods may speed or aid human investigation.
-More work is needed to address concerns related to interpretability and how to best model each problem.
+Deep learning, which describes a class of machine learning algorithms focussing on the training of deep artificial neural networks, has recently shown impressive results across a variety of domains.
+Biology and medicine are data-rich disciplines, but the data are complex and often ill-understood. Hence, deep learning techniques may be particularly well-suited to solve problems of these fields.
+We examine applications of deep learning to a variety of biomedical problems---patient classification, fundamental biological processes, and treatment of patients---and discuss whether deep learning will be able to transform these tasks or if the biomedical sphere poses unique challenges.
+As a result from an extensive literature review, we find that deep learning has yet to revolutionize biomedicine or definitively resolve any of the most pressing challenges in the field, but promising advances have been made on the prior state of the art.
+Even though improvements over previous baselines have been modest, the recent progress indicates that deep learning methods will provide valuable means for speeding up or aiding human investigation.
+However, deep learning models are still regarded as black box algorithms, and more work is needed to address the common concerns related to interpretability and how to best model each problem.
```

Comment:
I'm a bit reluctant to use the terminology `black box` here. I feel it's overused and under-justified, when many other methods that don't get this terminology applied have similar issues. What about:

> Though progress has been made in determining the primary factors that lead a specific deep neural network to make a specific prediction in a certain case, understanding how users should interpret these models to make specific mechanistic hypotheses remains an open challenge.

Reply:
☝️ I'm not a huge fan of that, but I feel like the sentiment is what i'm going for so I'll put it there.

Reply:
I like the sentiment as well, but `determining the primary factors that lead a specific deep neural network to make a specific prediction in a certain case` is quite verbose

Reply:
What about this:

> Though progress has been made linking a specific neural networks prediction to input features, understanding how users should interpret these models to make testable hypotheses about the system under study remains an open challenge.

Reply:
👍 but add `'` in `network's`